dvc/fs/azure.py

```diff
@@ -133,3 +164,9 @@ def fs(self):
             ) from e
 
         return file_system
+
+    def open(
+        self, path_info, mode="r", **kwargs
+    ):  # pylint: disable=arguments-differ
+        with _temp_event_loop():
```

Comment:
Could you elaborate why we need another temp event loop here again, please?

Reply:
See: https://github.com/iterative/dvc/pull/5958#issuecomment-832521073

Reply:
Sorry, blinded that one :facepalm: So I guess we are waiting for updates on https://github.com/dask/adlfs/issues/218 and then will maybe remove it. Gotcha.